integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/preaaggregate/PreAggregateTableHelper.scala

```diff
@@ -116,14 +116,33 @@ case class PreAggregateTableHelper(
         parentTable.getTableInfo.getFactTable.getTableProperties.asScala
           .getOrElse(CarbonCommonConstants.LOCAL_DICTIONARY_THRESHOLD,
             CarbonCommonConstants.LOCAL_DICTIONARY_THRESHOLD_DEFAULT))
-    tableProperties
-      .put(CarbonCommonConstants.LOCAL_DICTIONARY_INCLUDE,
-        parentTable.getTableInfo.getFactTable.getTableProperties.asScala
-          .getOrElse(CarbonCommonConstants.LOCAL_DICTIONARY_INCLUDE, ""))
-    tableProperties
-      .put(CarbonCommonConstants.LOCAL_DICTIONARY_EXCLUDE,
-        parentTable.getTableInfo.getFactTable.getTableProperties.asScala
-          .getOrElse(CarbonCommonConstants.LOCAL_DICTIONARY_EXCLUDE, ""))
+    val parentdict_include = parentTable.getTableInfo.getFactTable.getTableProperties.asScala
```

Comment:
need to optimize the variable's name

same problem in belowing lines

Reply:
done